jmxtrans-core/src/main/java/com/googlecode/jmxtrans/jobs/ServerJob.java

```diff
@@ -41,8 +47,17 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
 
 		JMXConnector conn = null;
 		JMXConnectionParams connectionParams = null;
+
 		try {
-			connectionParams = new JMXConnectionParams(server.getJmxServiceURL(), server.getEnvironment());
+			JMXServiceURL jmxUrl;
```

Comment:
LocalJob doesn't exist. This comment is incompatible with the one on getJmxServiceURL. You need to pick one way.